test/workbox-build/node/lib/copy-workbox-sw.js

```diff
@@ -0,0 +1,46 @@
+const expect = require('chai').expect;
```

Comment:
Most of the logic in the unit tests have stayed the same, with the exception of removing the tests for bad parameters.

I'm assuming that all usage will go through one of the `entry-point/` modules, and they handle validation.